Makefile

```diff
@@ -181,7 +198,14 @@ generate-doc: ## Generate the Rust documentation
 		@echo ""
 		@echo "Generating the documentation."
 		@echo ""
-		RUSTDOCFLAGS="-D warnings" cargo doc --all-features --no-deps --document-private-items --workspace --exclude xtask
+		RUSTDOCFLAGS="--enable-index-page -Zunstable-options" cargo +nightly doc \
+			--all-features \
+			--no-deps \
+			--document-private-items \
+			--workspace \
+			--exclude plonk_neon \
+			--exclude plonk_wasm \
+			--exclude xtask
```

Comment:
Why is xtask excluded? I don't think there are docs here, but there's nothing gained by excluding it